debian/rules

```diff
@@ -1,12 +1,9 @@
 #!/usr/bin/make -f
 
-# export DH_VERBOSE=1
-
-DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
```

Comment:
This variable was not used in the script, so setting it was redundant (and is now deprecated anyway, in favor of including `/usr/share/dpkg/architecture.mk` instead).